<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingsForm</class>
 <widget class="QWidget" name="SettingsForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>550</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Settings</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="0">
    <widget class="QPushButton" name="saveButton">
     <property name="text">
      <string>Save</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="general">
      <attribute name="title">
       <string>General</string>
      </attribute>
      <widget class="QCheckBox" name="condensedViews">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>90</y>
         <width>400</width>
         <height>32</height>
        </rect>
       </property>
       <property name="toolTip">
        <string>As an example, &quot;135,230 views&quot; would be condensed to &quot;135.2k views.&quot;</string>
       </property>
       <property name="text">
        <string>Use condensed view counts</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="preferLists">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>130</y>
         <width>400</width>
         <height>32</height>
        </rect>
       </property>
       <property name="text">
        <string>Prefer list layouts over grid layouts</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="returnDislikes">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>400</width>
         <height>32</height>
        </rect>
       </property>
       <property name="text">
        <string>Enable dislike counts</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QCheckBox" name="fullSubs">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>50</y>
         <width>400</width>
         <height>32</height>
        </rect>
       </property>
       <property name="text">
        <string>Use full subscriber counts</string>
       </property>
      </widget>
      <widget class="QLabel" name="appStyleLabel">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>170</y>
         <width>170</width>
         <height>26</height>
        </rect>
       </property>
       <property name="text">
        <string>Application style</string>
       </property>
      </widget>
      <widget class="QComboBox" name="appStyle">
       <property name="geometry">
        <rect>
         <x>180</x>
         <y>168</y>
         <width>135</width>
         <height>34</height>
        </rect>
       </property>
       <item>
        <property name="text">
         <string>Default</string>
        </property>
       </item>
      </widget>
      <widget class="QCheckBox" name="darkTheme">
       <property name="geometry">
        <rect>
         <x>330</x>
         <y>170</y>
         <width>93</width>
         <height>26</height>
        </rect>
       </property>
       <property name="text">
        <string>Dark</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="player">
      <attribute name="title">
       <string>Player</string>
      </attribute>
      <widget class="QCheckBox" name="restoreAnnotations">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>50</y>
         <width>400</width>
         <height>32</height>
        </rect>
       </property>
       <property name="text">
        <string>Restore classic video annotations</string>
       </property>
      </widget>
      <widget class="QLabel" name="preferredVolumeLabel">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>210</y>
         <width>170</width>
         <height>26</height>
        </rect>
       </property>
       <property name="text">
        <string>Preferred volume</string>
       </property>
      </widget>
      <widget class="QSpinBox" name="preferredVolume">
       <property name="geometry">
        <rect>
         <x>180</x>
         <y>208</y>
         <width>70</width>
         <height>34</height>
        </rect>
       </property>
       <property name="maximum">
        <number>100</number>
       </property>
       <property name="value">
        <number>100</number>
       </property>
      </widget>
      <widget class="QLabel" name="preferredQualityLabel">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>250</y>
         <width>170</width>
         <height>26</height>
        </rect>
       </property>
       <property name="text">
        <string>Preferred quality</string>
       </property>
      </widget>
      <widget class="QComboBox" name="preferredQuality">
       <property name="geometry">
        <rect>
         <x>180</x>
         <y>248</y>
         <width>135</width>
         <height>34</height>
        </rect>
       </property>
       <item>
        <property name="text">
         <string>Auto</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>4320p</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>2160p</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>1440p</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>1080p</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>720p</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>480p</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>360p</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>240p</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>144p</string>
        </property>
       </item>
      </widget>
      <widget class="QCheckBox" name="h264Only">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>90</y>
         <width>400</width>
         <height>32</height>
        </rect>
       </property>
       <property name="text">
        <string>Use H.264 only where possible</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="disable60Fps">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>130</y>
         <width>400</width>
         <height>32</height>
        </rect>
       </property>
       <property name="text">
        <string>Disable &gt;30 FPS video</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="disablePlayerInfoPanels">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>170</y>
         <width>400</width>
         <height>32</height>
        </rect>
       </property>
       <property name="text">
        <string>Disable info panels</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="blockAds">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>400</width>
         <height>32</height>
        </rect>
       </property>
       <property name="toolTip">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Videos generally will not have ads already because they are embedded, but there are some exceptions.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="text">
        <string>Block ads</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="privacy">
      <attribute name="title">
       <string>Privacy</string>
      </attribute>
      <widget class="QCheckBox" name="playbackTracking">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>400</width>
         <height>32</height>
        </rect>
       </property>
       <property name="toolTip">
        <string>Playback tracking allows the user to add to a video's view count when the video is played.</string>
       </property>
       <property name="text">
        <string>Enable playback tracking</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QCheckBox" name="watchtimeTracking">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>50</y>
         <width>400</width>
         <height>32</height>
        </rect>
       </property>
       <property name="toolTip">
        <string>Watch time tracking allows the History tab to function and is used in creator analytics.</string>
       </property>
       <property name="text">
        <string>Enable watch time tracking</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="filtering">
      <attribute name="title">
       <string>Filtering</string>
      </attribute>
      <widget class="QCheckBox" name="hideShorts">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>400</width>
         <height>32</height>
        </rect>
       </property>
       <property name="text">
        <string>Filter shorts</string>
       </property>
      </widget>
      <widget class="QPushButton" name="showFilteredChannels">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>130</y>
         <width>170</width>
         <height>35</height>
        </rect>
       </property>
       <property name="text">
        <string>Filtered Channels</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="hideStreams">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>50</y>
         <width>400</width>
         <height>32</height>
        </rect>
       </property>
       <property name="text">
        <string>Filter live streams</string>
       </property>
      </widget>
      <widget class="QPushButton" name="showFilteredTerms">
       <property name="geometry">
        <rect>
         <x>190</x>
         <y>130</y>
         <width>170</width>
         <height>35</height>
        </rect>
       </property>
       <property name="text">
        <string>Filtered Terms</string>
       </property>
      </widget>
      <widget class="QWidget" name="horizontalLayoutWidget">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>90</y>
         <width>311</width>
         <height>36</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QCheckBox" name="filterLengthCheck">
          <property name="text">
           <string>Filter videos ≤</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="filterLength">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="maximum">
           <number>5000</number>
          </property>
          <property name="singleStep">
           <number>10</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="filterLengthLabel">
          <property name="text">
           <string>seconds</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
     <widget class="QWidget" name="sponsorblock">
      <attribute name="title">
       <string>SponsorBlock</string>
      </attribute>
      <widget class="QLabel" name="blockLabel">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>400</width>
         <height>32</height>
        </rect>
       </property>
       <property name="text">
        <string>Block:</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="blockSponsor">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>40</y>
         <width>210</width>
         <height>32</height>
        </rect>
       </property>
       <property name="text">
        <string>Sponsors</string>
       </property>
       <property name="checked">
        <bool>false</bool>
       </property>
      </widget>
      <widget class="QCheckBox" name="blockIntro">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>70</y>
         <width>210</width>
         <height>32</height>
        </rect>
       </property>
       <property name="text">
        <string>Intros</string>
       </property>
       <property name="checked">
        <bool>false</bool>
       </property>
      </widget>
      <widget class="QCheckBox" name="blockOutro">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>100</y>
         <width>210</width>
         <height>32</height>
        </rect>
       </property>
       <property name="text">
        <string>Endcards/Credits</string>
       </property>
       <property name="checked">
        <bool>false</bool>
       </property>
      </widget>
      <widget class="QCheckBox" name="blockSelfPromo">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>130</y>
         <width>210</width>
         <height>32</height>
        </rect>
       </property>
       <property name="text">
        <string>Self Promotion</string>
       </property>
       <property name="checked">
        <bool>false</bool>
       </property>
      </widget>
      <widget class="QCheckBox" name="blockPreview">
       <property name="geometry">
        <rect>
         <x>250</x>
         <y>40</y>
         <width>224</width>
         <height>32</height>
        </rect>
       </property>
       <property name="text">
        <string>Previews/Recaps</string>
       </property>
       <property name="checked">
        <bool>false</bool>
       </property>
      </widget>
      <widget class="QCheckBox" name="blockFiller">
       <property name="geometry">
        <rect>
         <x>250</x>
         <y>70</y>
         <width>224</width>
         <height>32</height>
        </rect>
       </property>
       <property name="text">
        <string>Filler Tangents/Jokes</string>
       </property>
       <property name="checked">
        <bool>false</bool>
       </property>
      </widget>
      <widget class="QCheckBox" name="blockInteraction">
       <property name="geometry">
        <rect>
         <x>250</x>
         <y>100</y>
         <width>224</width>
         <height>32</height>
        </rect>
       </property>
       <property name="text">
        <string>Interaction Reminders</string>
       </property>
       <property name="checked">
        <bool>false</bool>
       </property>
      </widget>
      <widget class="QCheckBox" name="blockNonMusic">
       <property name="geometry">
        <rect>
         <x>250</x>
         <y>130</y>
         <width>224</width>
         <height>32</height>
        </rect>
       </property>
       <property name="text">
        <string>Non-Music Sections</string>
       </property>
       <property name="checked">
        <bool>false</bool>
       </property>
      </widget>
      <widget class="QCheckBox" name="showSBToasts">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>170</y>
         <width>400</width>
         <height>32</height>
        </rect>
       </property>
       <property name="text">
        <string>Show toasts (popups, notifications)</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="deArrowTab">
      <attribute name="title">
       <string>DeArrow</string>
      </attribute>
      <widget class="QCheckBox" name="deArrow">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>400</width>
         <height>32</height>
        </rect>
       </property>
       <property name="toolTip">
        <string>DeArrow is an open source browser extension for crowdsourcing better titles and thumbnails on YouTube.</string>
       </property>
       <property name="text">
        <string>Enable 'DeArrow' integration</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="deArrowTitles">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>50</y>
         <width>400</width>
         <height>32</height>
        </rect>
       </property>
       <property name="text">
        <string>Replace titles</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QCheckBox" name="deArrowThumbs">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>90</y>
         <width>400</width>
         <height>32</height>
        </rect>
       </property>
       <property name="text">
        <string>Replace thumbnails</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="about">
      <attribute name="title">
       <string>About</string>
      </attribute>
      <widget class="QLabel" name="qttubeLogo">
       <property name="geometry">
        <rect>
         <x>55</x>
         <y>5</y>
         <width>357</width>
         <height>80</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QLabel" name="description">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>100</y>
         <width>474</width>
         <height>164</height>
        </rect>
       </property>
       <property name="text">
        <string>TextLabel</string>
       </property>
       <property name="textFormat">
        <enum>Qt::RichText</enum>
       </property>
       <property name="alignment">
        <set>Qt::AlignHCenter|Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
       </property>
       <property name="openExternalLinks">
        <bool>true</bool>
       </property>
      </widget>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
